### PR TITLE
Improve object rest/spread with Proxy coverage

### PR DESCRIPTION
**Pull request by shvaikalesh**

### CODE REVIEW

File: test/language/expressions/object/object-spread-proxy-ownkeys-no-excluded-keys.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
```
Comment (rwaldron) [inline test/language/expressions/object/object-spread-proxy-ownkeys-no-excluded-keys.js:1]: 2021 ;)

Comment (shvaikalesh) [reply test/language/expressions/object/object-spread-proxy-ownkeys-no-excluded-keys.js]: Fixed, thanks.

I hope you had a pleasant holiday!

**Comment by shvaikalesh** [comment]
_cc_ @syg: V8 seems to call `[[GetOwnProperty]]` on excluded names during object rest destructuring.

**Review by jugglinmike — APPROVED** [review]
Nice work, @shvaikalesh!

**Pull request merged**